advice-dont-wait-to-ask.md

```diff
@@ -0,0 +1,48 @@
+# Don't wait to ask[^1]
+
+You may have been sent this document because **you waited to ask** a question,
+such as:
+
+> You: Hi<br/>
+> You: Anyone got a second?<br/>
+> You: Quick question.<br/>
+> You: `@someone`?
+
+In asynchronous messaging, this is similar to calling someone, then putting them on hold
+after they pick up. If you prompt someont to respond before asking your question, you won't
+get an answer sooner but are instead asking for someone's attention.
+
+> [!TIP]
+> **tl;dr:** don't wait to ask your question or wait for someone to respond, but rather
+> write your question immediately.
+
+This causes conflict and problems because:
+- no one may respond, since they can't prioritize your message
+- someone may respond while you won't be there to ask your question
+- someone is prompted to give you their full attention to wait for your question
+
+Maybe you tried to be polite and wanted to ease someone into your question or introduce
+yourself before jumping into a request or asking for someone to commit to helping you - like
+one would in person or on the phone? That's great!
+But, online chats and forums are different.
+
+Instead of letting someone respond in their own time, you're making them acknowledge your
+message before then making them wait for you to phrase your question, which is lost time for
+both you and the person responding (and that's frustrating for everyone).
```

Comment:
```suggestion
By asking to ask, instead of letting someone respond in their own time,
you're making them acknowledge your message first, then wait for you to phrase your question,
which is lost time for both you and the person responding (and that's frustrating for everyone).
```